pkg/network/ovn_kubernetes.go

```diff
@@ -288,6 +288,12 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		data.Data["OVN_GATEWAY_MODE"] = OVN_SHARED_GW_MODE
 	}
 
+	if c.EgressIPConfig.ReachabilityTotalTimeoutSeconds != nil {
+		data.Data["ReachabilityTotalTimeoutSeconds"] = *c.EgressIPConfig.ReachabilityTotalTimeoutSeconds
+	} else {
+		data.Data["ReachabilityTotalTimeoutSeconds"] = 1
```

Comment:
where we will set the default ? api folks didn't want me to use kubebuilder default in the api

Reply:
this won't work because we need to allow user to configure 0 to disable the check, the above suggested change will fail unit-test I added to disable reachability check